packages/opentelemetry-plugin-express/README.md

```diff
@@ -0,0 +1,78 @@
+# OpenTelemetry Express Instrumentation for Node.js
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`express`](https://github.com/expressjs/express).
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-node) package.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/plugin-express
+```
+### Supported Versions
+ - `^4.0.0`
+
+## Usage
+
+OpenTelemetry Express Instrumentation allows the user to automatically collect trace data and export them to their backend of choice, to give observability to distributed systems.
+
+To load a specific plugin (express in this case), specify it in the Node Tracer's configuration.
+```js
+const { NodeTracerProvider } = require('@opentelemetry/node');
+
+const provider = new NodeTracerProvider({
+  plugins: {
+    express: {
+      enabled: true,
+      // You may use a package name or absolute path to the file.
+      path: '@opentelemetry/plugin-express',
+    }
+  }
+});
+```
+
+To load all the [supported plugins](https://github.com/open-telemetry/opentelemetry-js#plugins), use below approach. Each plugin is only loaded when the module that it patches is loaded; in other words, there is no computational overhead for listing plugins for unused modules.
+```js
+const { NodeTracerProvider } = require('@opentelemetry/node');
+
+const provider = new NodeTracerProvider();
+```
+
+See [examples/express](https://github.com/open-telemetry/opentelemetry-js/tree/master/examples/express) for a short example.
```

Comment:
This link is not present.

Reply:
I've done the same with `mongodb`, its just to avoid updating the readme when adding a exemple.

Reply:
I would prefer not to have broken links and add them when there is actually something to point to